### PR TITLE
fix(grid-manager): fix height re-render

### PR DESCRIPTION
**Pull request by felipeegasneris**
WIP

### CODE REVIEW

**Comment by felipeegasneris** [comment]
[(5) bug input text en mobile](https://trello.com/c/yjcrcKwr/312-5-bug-input-text-en-mobile)

File: packages/grid-manager/src/grid/utils.ts [hunk 1]
```diff
@@ -97,6 +98,13 @@ export const getGridPositions = (
   };
 };
 
+export const getHeightRulesFromViewport = (
```
Comment (alex-sysoev) [inline packages/grid-manager/src/grid/utils.ts:101]: Falta poner el tipo del valor que retorna la función, probablemente sera Boolean si entiendo bien, de acá sale la pregunta sobre el nombre de la función que se supone devuelve Rules cómo la otra mas abajo.

File: packages/grid-manager/src/index.ts [hunk 2]
```diff
@@ -113,14 +112,15 @@ class GridManager implements GridManagerProps {
...
     this.interpreter = interpret(this.widgetMachine, { devTools: true })
-      // .onTransition((state, event) => {
-      //   // todo add log service
-      //   // event type: ${event.type}`
-      // })
-      // .onDone((state) => {
-      //   // todo add log service
-      //   // `reach final state`, { state }
-      // })
+      .onTransition((state, event) => {
```
Comment (alex-sysoev) [inline packages/grid-manager/src/index.ts:115]: Remove logs please.

File: packages/grid-manager/src/widgetsMachine/actions.ts [hunk 3]
```diff
@@ -241,6 +250,15 @@ export const reconcileWidgets = (context: WidgetsMachineCtx) => {
...
   };
 
+  const toUpdate =
+    requireHeightUpdate && !requireGlobalUpdate
+      ? getWidgetsShapeToUpdate(
+          context.renderCycle.widgetsInDom,
+          context.widgets,
+          activeBreakpoint
+        )
+      : context.renderCycle.updateCycle.update;
```
Comment (alex-sysoev) [inline packages/grid-manager/src/widgetsMachine/actions.ts:260]: Por que tu asignas el update cycle a heightUpdate si no requiere height update ?

**Review by alex-sysoev — CHANGES_REQUESTED** [review]
PR is very difficult to review. When reviewing from `Files changed` all your changes are mixed with formatter and linter changes and are disordered. When reviewing commit by commit it is almost impossible because of multiple changes of the same files. Please use rebase interactive to order your PR o think about order and PR clarity beforehand.

**Pull request merged**